lib/views/layout.erb

```diff
@@ -0,0 +1,22 @@
+<head>
+  <h1> Chitter </h1>
+  <h3> <a href="/sessions/new">Sign in</a> <a href="/users/new">Sign up</a> <a href="/">Home</a> </h3>
+</head>
+<body>
+  <% if current_user %>
+    Welcome, <%= current_user.email %>
+    <form method="post" action="/sessions">
+      <input type="hidden" name="_method" value="delete">
+      <input type="submit" value="Sign out">
+    </form>
+  <% end %>
+  <% if flash[:errors] && !flash[:errors].empty? %>
```

Comment:
you are only displaying flash[:error] here and not flash[:notice] - this is why your test is failing